hpack/hpack.py

```diff
@@ -362,15 +366,20 @@ def _encode_literal(self, name, value, indexing, huffman=False):
             name_len[0] |= 0x80
             value_len[0] |= 0x80
 
-        return b''.join([prefix, bytes(name_len), name, bytes(value_len), value])
+        return b''.join([indexbit, bytes(name_len), name, bytes(value_len), value])
 
-    def _encode_indexed_literal(self, index, value, huffman=False):
+    def _encode_indexed_literal(self, index, value, indexbit=INDEX_INCREMENTAL, huffman=False):
```

Comment:
I think I mostly did it as a safety to start.  But yeah there's no point in it now. Same might be said of huffman too actually, I'd have to look.